src/styles/auth-module.scss

```diff
@@ -160,6 +160,52 @@
         }
       }
     }
+
+    //Register
+    .register-view-form-wrapper{
+      margin: 0 auto;
+      @include auth-form-wrapper;
+      .heading{
+        padding-top: $default-padding*2;
+        padding-bottom: $default-padding*3;
+        @include screen-heading;
+      }
+      .ui.form{
+        .btn-wrapper{
+          margin-top: $default-margin*4;
+          margin-bottom: $default-margin*2;
+          @include align-item(flex, space-between, center, row);
+          .app-btn{
+            @include app-btn(100px, 50px, $theme-color, $theme-color);
+          }
+        }
+        .register-footer-wrapper {
+          display: flex;
```

Comment:
We can make the use of already created mixin `align-item` here. Check `mixins.scss` for the reference.